test/volume_test.cpp

```diff
@@ -37,7 +40,7 @@ void test_volume(FilePath f, NT expected, NT tolerance=0.1)
     boost::random::uniform_real_distribution<>(urdist);
     boost::random::uniform_real_distribution<> urdist1(-1,1);
     
-    vars var(rnum,n,walk_len,n_threads,err,e,0,0,0,0,rng,
+    vars<NT, RNGType> var(rnum,n,walk_len,n_threads,err,e,0,0,0,0,rng,
              urdist,urdist1,-1.0,false,false,false,false,false,false,true);
 
     //Compute chebychev ball//
```

Comment:
It would be useful to also print the number type otherwise it is not easy to identify which test is running. You can use `typeid`, see https://en.cppreference.com/w/cpp/language/typeid